question-5/question-5.php

```diff
@@ -7,11 +7,32 @@
  * 1. Register custom post type
  */
 
-// Your code here
+// Register Custom Post Type
+function lion_interview() {
+  $args = array(
+    'public'                => false,
+    'show_ui'               => true,
+    'show_in_menu'          => true,
+  );
+  register_post_type( 'post', $args );
```

Comment:
This is registering a new post type called `post` (which will throw an error, since `post` is of course on of the built-in post types).
```suggestion
  register_post_type( 'lion__interview', $args );
```

Reply:
@bryanatlion same issue here with a missing commit. Resolved.